### DISCUSSION TITLE
Change light default slider

### DISCUSSION THREAD

**Discussion by SavuBogdan** [piitaya/lovelace-mushroom#918] [Q&A]
Is there any way to default to color slider instead of brightness slider on page refresh ?

i see this is default to the first entry in the array https://github.com/piitaya/lovelace-mushroom/blob/main/src/cards/light-card/light-card.ts#L157 
any way to change that behaviour ?

**Comment by costa-alex** [comment]
I’m interested in this as well. If possible, maybe add an option to choose the default slider?

thanks
